website/docs/community/contribute.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Contributing
+id: contribute
+---
+
+import DocCardList from '@theme/DocCardList';
+import {useCurrentSidebarCategory} from '@docusaurus/theme-common';
+
+## Want to get involved? Start here
+
+The dbt Community predates dbt Labs as an organization and harkens back to the days when a scrappy analytics consultancy of a few [pissed off data analysts](<https://www.hashpath.com/2020/12/an-analytics-engineer-is-really-just-a-pissed-off-data-analyst/#:~:text=Often%20times%2C%20an%20analytics%20engineer,necessity%20(and%20genius%20branding).>) started hacking together an open source project around which gathered a community that would change how the world uses data.
+
+[The dbt Community](https://www.getdbt.com/community/) exists to allow analytics practitioners to share their knowledge, help others and collectively to drive forward the discipline of analytics engineering. This is something that can’t be done by any one individual or any one organization - to create a new discipline is necessarily a community effort. The **only** reason that dbt has become as widespread as it has is because people like you choose to get involved and share your knowledge. Contributing to the community can also be a great way to learn new skills, build up a public portfolio and make friends with other practitioners.
+
+There are opportunities here for everyone to get involved. Whether you are just beginning your analytics engineering journey or you are a seasoned data professional - contributing isn’t about knowing all of the answers, it’s about learning things together.
+
+Below - you’ll find a sampling of the ways to get involved. It might feel complicated or like there are a lot of options - but these are ultimately just variations on the theme of sharing knowledge with the broader community.
+
+<DocCardList items={useCurrentSidebarCategory().items} />
```

Comment:
is this basically a way to do an auto-generated "category" page but actually have a more fleshed-out Markdown intro rather than just a simple string?

Reply:
yep exactly! took some digging to figure it out but that's precisely it.